bird_Docker_build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9
+
+WORKDIR /
+
+RUN wget https://github.com/stan-dev/cmdstan/releases/download/v2.30.0/cmdstan-2.30.0.tar.gz
+RUN tar -zxpf cmdstan-2.30.0.tar.gz
+
+RUN mv cmdstan-2.30.0 cmdstan
+WORKDIR /cmdstan
+RUN make build
+
+RUN wget --no-check-certificate https://github.com/bmajoros/BIRD/blob/master/bird.py
```

Comment:
Is `--no-check-certificate` necessary, or is it just to get around Zscaler issues? If it's just to get around Zscaler issues, I think you should remove it.